src/encoding/armor.js

```diff
@@ -246,7 +244,7 @@ function splitChecksum(text) {
 
   if (lastEquals >= 0) {
```

Comment:
Add the condition 'lastEquals !== text.length - 1' here? Otherwise in the case of missing checksum a = gets stripped from the end of the body.